### DISCUSSION TITLE
play.etcd.io??

### DISCUSSION THREAD

**Discussion by Cloud-Mak** [etcd-io/etcd#19014] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/e17edc35-9c2f-48d2-b3b9-4e7578196d51)

I could only see loading and nothing gets displayed in here.  I am new to etcd. Is this some kind of playground for practicing etcd?

**Comment by jmhbnz** [comment, marked as answer]
Thanks for your question @Cloud-Mak - Yes this is the website for repository https://github.com/etcd-io/etcdlabs.

Unfortunately there is an issue with it currently https://github.com/etcd-io/etcdlabs/issues/293 that we have not had capacity to address yet.

Edit:  I've restated the vm and it looks to be alive again (just use `http://` instead of `https://`).
